packages/modul-components/src/components/table/table.ts

```diff
@@ -68,6 +68,19 @@ export class MTable extends ModulVue {
     @Prop({ default: true })
     public rowHighlightedOnHover: boolean;
 
+    @Prop({
+        default: '100%',
+        validator: value => {
+            const pixelOrPercentageNumberRegExp: RegExp = /^\d+(\.[0-9]{1,4})?(px|%)$/;
+            if (!value.search(pixelOrPercentageNumberRegExp)) {
+                // tslint:disable-next-line: no-console
+                console.warn(`width-placeholder value needs to respect this RegEx: ${pixelOrPercentageNumberRegExp}`);
+            }
+            return !value.search(pixelOrPercentageNumberRegExp);
+        }
+    })
```

Comment:
De @vidal7 



```
width-placeholder value needs to respect this RegEx
```

Le validator ne valide pas la valeur par défaut qui est 100% l'expression régulière est ok mais le search retourne 0 et donc entre dans la condition "value.search. 

 Je pense que l'on aurait du faire un !value.match à la place

Reply:
Aussi utiliser un this.$log.warn